components/common-go/util/workspaceid_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package util_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gitpod-io/gitpod/common-go/util"
+)
+
+func TestGenerateWorkspaceID(t *testing.T) {
+
+	t.Run(fmt.Sprintf("check names are valid"), func(t *testing.T) {
```

Comment:
No need for the `t.Run`. The outer `t` is already a valid test. The `t.Run` would indicate a sub test,